packages/axelar-wasm-std/src/utils.rs

```diff
@@ -62,3 +62,13 @@ impl<T, E> InspectorResult<T, E> for Result<T, E> {
         })
     }
 }
+
+pub trait TryMapExt<T, B, E> {
```

Comment:
Why not just `OptionExt` so that more methods can be added later to the same trait?

Reply:
I'm undecided if I'd rather keep those in separate traits or collect them all, so I'm deferring that decision to the future for now